Update `BBCode` (#7901)
ppy/osu-wiki: wiki/BBCode/en.md

*Note: The user ID is a string of numbers that directly follow the `/users/` in the URL of an osu! profile page.*

> If specified through their user ID, the text between the open and closed tags will not be parsed and will instead display as the user's current username. In case there is no text between the open and closed tags specified, the link to the user profile is not generated correctly. However, if the profile page is specified only through a username, and said user changes their username, the link will cease to function.

Reviewer: ```suggestion
If specified through their user ID, the text between the open and closed tags will not be parsed and will instead display as the user's current username. In case there is no text between the open and closed tags specified, the link to the user profile will not be generated correctly. However, if the profile page is specified only through a username, and said user changes their username, the link will cease to function.
```

> If specified through their user ID, the text between the open and closed tags will not be parsed and will instead display as the user's current username. In case there is no text between the open and closed tags specified, the link to the user profile will not be generated correctly. However, if the profile page is specified only through a username, and said user changes their username, the link will cease to function.

Reviewer 2: Here for JP>EN translation check. I do not have any knowledge about the topic so please feel free to change it.

When specifying users only by their user IDs, a placeholder text is required between the open and closed tags. Doing so will display the username on the actual page, not the placeholder text. When specifying users only by their username, the link will not function once they change their username.

Author: ```suggestion
When specifying users only by their user IDs, a placeholder text is required between the open and closed tags. Doing so will display the username on the actual page, not the placeholder text. When specifying users only by their username, the link will not function once they change their username.
```

[/list]
```

The `[list]` tag is used to automatically format two different types of lists throughout the osu! forums by using an asterisk enclosed in brackets (`[*]`) to indicate a new item in the list (shown above). By default, this will create a plain, bulleted list.

> If the `TYPE` argument is specified as `1`, the list will be numbered.

Reviewer 3: It can also be numbered if a value other than "1" is specified

Reviewer 2: Made it similar to 202's end.

> If the `TYPE` argument is specified as `1`, it will create a numbered list.

Author: As mentioned in the description, I wanted the article to be understandable more easily.

> If the `TYPE` argument is specified as `1`, the list will be numbered.

Reviewer 2: Made it similar to 202's end.

> If the `TYPE` argument is specified as `1`, it will create a numbered list.

Author: ```suggestion
If the `TYPE` argument is specified as `1`, it will create a numbered list.
```

*Notice: BBCode-formatted lists can be stacked on top of each other and placed inside one another, although this has been known to cause issues with formatting.*

Toolbar buttons: ![List button](img/list.png "List") ![Numbered list button](img/list-numbered.png "Numbered list")

### Email

```
[email=ADDRESS]text[/email]
```

The `[email]` tag creates a clickable hyperlink, which opens a new email in the default mail program with the address field pre-populated.

> In order to create a link, two arguments need to be specified: the `ADDRESS` argument needs to be a valid email address, while `text` is the displayed text when clicking on the link. If the `text` argument is not defined, the hyperlink will not be created correctly.

Reviewer: ```suggestion
In order to create a link, two arguments need to be specified: the `ADDRESS` argument needs to be a valid email address, while `text` is the displayed text that serves as a hyperlink. If the `text` argument is not defined, the hyperlink will not be created correctly.
```
